fetch_local.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This file is for testing the app locally. It downloads the latest file without checking and unzips it into the data folder.